models/comments.model.js

```diff
@@ -0,0 +1,22 @@
+const db = require("../db/connection.js");
+
+exports.removeCommentById = (comment_id) => {
+
+    if(isNaN(comment_id) === true) {
```

Comment:
This is fine but I think it will be handled by psql and give you an error any way so I might have been tempted to handle that error rather than throw a new one, the argument for this would be that we are applying a piece of logic to all successful requests that are not necessary